.github/workflows/test-deployment.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: test-deployment
+
+on: pull_request
+jobs:
+  docker-build:
+    permissions:
+      contents: read
+      packages: write
+    uses: scality/workflows/.github/workflows/docker-build.yaml@v1
+    with:
+      name: runner-manager
+      namespace: scality
+  test-deployment:
+    needs: docker-build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: helm/kind-action@v1
+      - name: Edit the runner-manager image
+        working-directory: manifests/base/runner-manager
+        run: |
+          kustomize edit set image runner-manager=ghcr.io/scality/runner-manager:${{ github.sha }}
```

Comment:
Can the set be done within the build command?

Reply:
I don't think so. I have searched how to override the image but I could not find a solution. I thought about parsing the image tag while building but that does not work too.

Reply:
```suggestion
          kustomize edit set image ghcr.io/scality/runner-manager=ghcr.io/scality/runner-manager:${{ github.sha }}
```

Reply:
it is not working as expected. this is because i run the edit before deploying which I do through kustomize build